compiler/rustc/Cargo.toml

```diff
@@ -4,10 +4,6 @@ name = "rustc-main"
 version = "0.0.0"
 edition = '2018'
 
-[[bin]]
-name = "rustc_binary"
-path = "src/main.rs"
```

Comment:
Bootstrap actually assumes the name of the binary is `rustc_binary`, so you also need to change the package name on line 3 to `rustc_binary`, I think.